### ISSUE TITLE
Image upload

### ISSUE THREAD

**Issue by Hagethorn** [themeskult/wp-svbtle#40]
Hi,

Love wp-svbtle so far. One thing bugs me is that there is no place to upload images while in the /wp-svbtle admin area. Did I miss anything? Going to  the /wp-admin area just to upload an image is kinda annoying. Hotlinking images or FTP's uploads are not efficient either. Would love to see it all integrated into the early clean admin environment.

Maybe there can be something in the posts' "Option" area together with the External URL for this?

**Comment by joshkerr** [comment]
I don't agree.  There are so many ways to get an image hosted online without needing Wordpress to do it.  Adding an "insert image" feature would add clutter to the clean UX.

**Comment by rickyrauch** [comment]
@joshkerr totally agree.

**Comment by rickyrauch** [comment]
Reopened 

http://blog.alexmaccaw.com/svbtle-image-uploading

**Comment by joshkerr** [comment]
Okay, I changed my mind, go for it.

**Comment by Hagethorn** [comment]
If you want to provide a clean UX, you can't leave out uploading images. It's like having to download a mail app from the app store before you can view your mail on an iPhone. Less is not always more.

**Comment by rickyrauch** [comment]
@joshkerr 
I = WE
my = OUR

**Comment by joshkerr** [comment]
Hook it up @ricardorauch, the guy basically gave you the source code.   Easy-as-pie.

**Comment by maccman** [comment]
Hah

**Comment by joshkerr** [comment]
@maccman well, you did give it to him, and the rest of the world.  Nicely done BTW.

**Comment by awelfle** [comment]
This _is_ awesome! ... Now I have some learning ahead of me to figure out how to implement this. For someone with a higher-than-average-but-still-much-less-than-professional understanding of JQuery, this is basically in Greek.

**Comment by dasrecht** [comment]
Any progress on this one? I guess @ricardorauch included it in the pay version of the wp-svbtle editor.

@ricardorauch ist there hope that the opensource version gets the love of image uploading too?
